src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/ProvidersController.cs

```diff
@@ -117,6 +127,61 @@ public void Head(long ukprn)
             Get(ukprn);
         }
 
+        /// <summary>
+        /// Get list of active apprenticeships for a given provider
+        /// </summary>
+        /// <param name="ukprn">UKPRN</param>
+        /// <returns>A list of active apprenticeships sorted by name alphabetically, then type, then level</returns>
+        [SwaggerResponse(HttpStatusCode.OK, "OK", typeof(IEnumerable<ApprenticeshipTraining>))]
+        [SwaggerResponse(HttpStatusCode.InternalServerError)]
+        [SwaggerResponse(HttpStatusCode.BadRequest, BadUkprnMessage)]
+        [Route("providers/{ukprn}/active-apprenticeship-training", Name = "GetActiveApprenticeshipsByProvider")]
+        [ExceptionHandling]
+        public ApprenticeshipTrainingSummary GetActiveApprenticeshipTrainingByProvider(long ukprn)
+        {
```

Comment:
maybe a refactor extracting some logic to a different method?

Reply:
now done